hphp/zend/html_tables/generate-html-table.py

```diff
@@ -190,7 +190,7 @@ def make_multicode_table(file_name, table_name):
 output_mapping('mappings/KOI8-R.TXT', 'entity_table_cs_koi8r')
 #output_mapping('mappings/ROMAN.TXT', 'entity_table_cs_macroman')
 
-# table for encoding csjis is esentially empty
+# table for encoding sjis is essentially empty
```

Comment:
I was debating between `cs-sjis` and `sjis` -- I think I eventually decided that other stuff favored `cs-sjis` but apparently I failed to go back and fix it.